{{cookiecutter.project_name}}/noxfile.py

```diff
@@ -254,9 +261,22 @@ def xdoctest(session: Session) -> None:
     session.run("python", "-m", "xdoctest", package, *args)
 
 
+@nox.session(name="docs-build", python="3.8")
+def docs_build(session: Session) -> None:
+    """Build the documentation."""
+    args = session.posargs or ["docs", "docs/_build"]
+    install(session, "sphinx")
+
+    builddir = Path("docs", "_build")
+    if builddir.exists():
+        shutil.rmtree(builddir)
+
+    session.run("sphinx-build", *args)
+
+
 @nox.session(python="3.8")
 def docs(session: Session) -> None:
-    """Build the documentation."""
+    """Interactive build for documentation."""
```

Comment:
Maybe something more explicit? Like this:

```suggestion
    """Build and serve the documentation, with live reloading on file changes."""
```

Reply:
Perfect =)